src/pages/iv-metals/index.js

```diff
@@ -0,0 +1,117 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useWindow } from 'src/windows'
+import MetalsWindows from './Window/MetalsWindow'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import { ControlContext } from 'src/providers/ControlContext'
+
+const TaxCodes = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { stack } = useWindow()
+
+  async function fetchGridData() {
+    const response = await getRequest({
+      extension: InventoryRepository.Metals.qry,
+      parameters: `_filter=`
+    })
+
+    return response
+  }
+
+  const {
+    query: { data },
+    labels: _labels,
+    invalidate,
+    refetch,
+    access
+  } = useResourceQuery({
+    queryFn: fetchGridData,
+    endpointId: InventoryRepository.Metals.page,
```

Comment:
here u are using page endpoint while above in fetchGridData u are using qry
let them call same endpoint